.github/workflows/fake-bpy-ci.yml

```diff
@@ -0,0 +1,84 @@
+name: fake-bpy-module CI
+
+on:
+  push:
+    branches:
+      # Temporarily trigger only on "ci_testing" branch
+      - ci_testing
+  # # Trigger on all PRs. By default, only run if activity type is opened, synchronize, or reopened
+  # pull_request:
+
+jobs:
+  build_modules:
+    name: Build fake-bpy modules
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.7"
+
+      - name: Blender downloads cache
+        uses: actions/cache@v2
+        id: blender-bin-cache
+        with:
+          path: blender-bin
+          key: blender-binaries-${{ runner.os }}-${{ hashFiles('**/tools/utils/download_blender.sh') }}
+
+      - name: Install python dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install wheel sphinx
+          pip install -r requirements.txt
+
+      - name: Install Blender binary dependencies
+        run: sudo apt-get install -y libglu1-mesa
+
+      - name: Install fake-bpy-module tools
+        run: sudo apt-get install -y pandoc
```

Comment:
It would be nicer to merge to L38 and rename `name` to `Install dependent packages` or similar.

Reply:
@nutti I kept them separate because of binary dependencies of blender change in a different rhythm (depending on Blender, out of our hands). Dependencies by fake-bpy is under full control of fake-bpy and can be changed easily.

I think it makes it easier for developers to decide if they need to add/remove packages from the L41, while they cannot decide anything about L38 (it depends on Blender).

What do you think?

Reply:
OK, then left it separated action.